packages/sdk/chainDeploy/helpers/types.ts

```diff
@@ -136,7 +136,8 @@ export type JarvisLiquidityPool = {
   liquidityPoolAddress: string;
 };
 
+// TODO remove?
 // future proofing for when we can deploy more than one liq pool liquidator
```

Comment:
@carlomazzaferro we don't need anymore to deploy multiple jarvis liquidators, because the different pools will be supplied through `strategyData`.. should we now remove `JarvisLiquidatorFunderDeployParams` and `deployJarvisLiquidatorFunder` ?

Reply:
nvm, removed it